### ISSUE TITLE
Maximum Plasma

### ISSUE THREAD

**Issue by Pestilenzius** [sparticle999/SpaceCompany#127]
I was thinking if it is intended that you can only accumulate 100.000 Plasma. It just seems odd being able to Energy-Mass convert only 33.333 Meteorite at once, considering this is a pretty low amount after even a single week of mediocre  checking on the game.
To be fair if you have set up enough plasma generating plants your meteorite generation should be pretty high, but sometimes you just want that extra meteorite from accumulated, but otherwise unused plasma.
Otherwise pretty balanved game and I have enjoyed it so far!

pestilenzius

**Comment by Pestilenzius** [comment]
a little edit: I generally feel that mass-energy conversion is rarely worth it at all. Maybe that is just my playstyle, but every machine I have outputs a better conversion rate, especially considering that uprading the batteries to store energy is pretty costly compared to uprgrading permanently working machines

**Comment by sparticle999** [comment]
I'm not sure how far you are in the game, but many players find EMC useful after their first few swarms

**Comment by Pestilenzius** [comment]
I am so close on building the whole Dyson sphere and I have already unlocked every wonder except for the portal room (which isn't an issue but I always keep forgetting to unlock it).
Maybe I just missed the phase where EMC is useful... Had a lot of trouble with the energy management and thus no energy to convert. But considering this is my first run at the game it all went considerably well.

EDIT: I just realised that the 100k limit on Plasma is acting like a break on progressing too fast by just buying all the meteorite. If that is intended, kudos to you. I like the pacing it creates.
